packages/runner/src/lib/dom.js

```diff
@@ -338,8 +342,8 @@ function isInViewport (win, el) {
   return (
     rect.top >= 0 &&
     rect.left >= 0 &&
-    rect.bottom <= win.innerHeight &&
-    rect.right <= win.innerWidth
+    rect.bottom <= $(win).height() &&
```

Comment:
I thought we wanted to use `outerHeight` and `outerWidth` here... ? Since that's the entire point of why we moved to use the native methods...

Can you add a test?